### ISSUE TITLE
Progress Indicator?

### ISSUE THREAD

**Issue by RVRover** [sneakypete81/updatescanner#124]
Does the new version show search progress?  I asked for "Search All Pages" (unfortunately I have hundreds, hence the request to be able to manually scan individual folders), but there is no indication that anything is happening and nothing has turned bold.

**Comment by sneakypete81** [comment]
Good idea - adding a progress indicator to the bottom of the sidebar should be possible, controlled via messages from the background process.

We'd probably need to make sure there's only ever one scan process running - if new page scans get requested while a scan is running, they should be appended to the running scan.

Another option would be to add a subtle animation to the toolbar icon when a scan is occurring.

**Comment by gunnicom** [comment]
Or maybe a progressbar under each site like: 
![370153eaf3dac3b973e131ac441b4b117ff11b33](https://user-images.githubusercontent.com/550251/32937274-19a2ed38-cb78-11e7-9344-52a70e216fe8.jpg)

Start a progressbar when the scan is triggered, so you can see which sites are about to scan.

**Comment by RVRover** [comment]
That would be even better. I have to be honest with you though. The ver. 4
add-on, in it's current form, is not something I can use and I've
downgraded.

Just so you know, here's how I use the add-on. I have hundreds (maybe even
over a thousand) of URLs bookmarked, but there are a couple of hundred I
check (manually) every morning. These are contained in folders. Without the
ability to initiate searches on individual folders, I would have to search
my entire, very large collection. Issue 2 is that because of the many URLs
I monitor I need to see which specific ones have changes (e.g. the bold
function). Just knowing the number of sites with changes is not helpful,
since I would still need to guess which of my many URLs actually have
recent changes. The absence of these features greatly compromises the
utility of the add-on for me.

**Comment by sneakypete81** [comment]
Understood, thanks for taking the time to report these issues, and it's good to hear that you find it useful. It was important to get the WebExtension update released before FF57 broke compatibility with the previous addon. But that means it doesn't yet have all the features I would like. Thanks for your patience.

**Comment by btanner2** [comment]
Progress Indicator would be great. Right now you are not able to see whats going no im update scanner started to work.

Thanks

Brian

Firefox: 58.0.2 (64-Bit)
Update Scanner: 4.2.0
